csrc/java_evp_keys.cpp

```diff
@@ -350,14 +350,15 @@ JNIEXPORT jbyteArray JNICALL Java_com_amazon_corretto_crypto_provider_EvpEdEcPri
         CHECK_OPENSSL(pkcs8.isInitialized());
 
         // This next line allocates memory
-        int bufLen = i2d_PKCS8_PRIV_KEY_INFO(pkcs8, &privateKeyBuffer);
-        CHECK_OPENSSL(bufLen > 0);
+        int bufInt = i2d_PKCS8_PRIV_KEY_INFO(pkcs8, &privateKeyBuffer);
+        CHECK_OPENSSL(bufInt > 0);
 
-        size_t bufLen2 = (size_t)bufLen;
-        CHECK_OPENSSL(EVP_PKEY_get_raw_private_key(key, privateKeyBuffer, &bufLen2) > 0);
+        size_t bufSize = (size_t)bufInt;
+        CHECK_OPENSSL(bufSize == (size_t)bufInt);
```

Comment:
Remove this, I misunderstood the code. I thought passing in the reference to `EVP_PKEY_get_raw_private_key` would return the bytes written and that would be useful to assert.